### PR TITLE
Generalizes CostSummary component

### PR DESCRIPTION
**Pull request by jessicamarcus**
- Refactored CostSummary component to receive the number of certificates for an order instead of a `DeathCertificateCart` instance
- Added birth cert costs, research fee
- Enhanced component’s `<table>` element for better a11y support
- Moved CSS to bottom of file for easier readability

### CODE REVIEW

File: services-js/registry-certs/client/common/CostSummary.tsx [hunk 1]
```diff
@@ -1,74 +1,36 @@
...
+  DEATH_CERTIFICATE_COST_STRING,
 } from '../../lib/costs';
 
-import DeathCertificateCart from '../store/DeathCertificateCart';
-
 type ServiceFeeType = 'CREDIT' | 'DEBIT';
 
 interface Props {
-  cart: DeathCertificateCart;
+  certificateType?: 'birth' | 'death';
```
Comment (fionawhim) [inline services-js/registry-certs/client/common/CostSummary.tsx:18]: 👍

File: services-js/registry-certs/client/common/CostSummary.tsx [hunk 2]
```diff
@@ -89,32 +51,65 @@ export default class CostSummary extends React.Component<Props, State> {
...
     return (
       <div className={CLEARFIX_STYLE}>
         <table className="t--info ta-r" style={{ float: 'right' }}>
+          <caption className={VISUALLYHIDDEN}>Cost Summary</caption>
+          <thead className={VISUALLYHIDDEN}>
+            <tr>
+              <th scope="col">Item</th>
+              <th scope="col">Amount</th>
+            </tr>
+          </thead>
```
Comment (fionawhim) [inline services-js/registry-certs/client/common/CostSummary.tsx:87]: nice!

File: services-js/registry-certs/lib/costs.ts [hunk 3]
```diff
@@ -3,62 +3,89 @@
...
-// CC == "credit card"
+// Per-transaction fee for records dated before 1870.
+export const RESEARCH_FEE = 10 * 100;
+
+// CC == “credit card”
 export const FIXED_CC_SERVICE_FEE = 25;
 export const PERCENTAGE_CC_SERVICE_FEE = 0.021;
 
-// DC == "debit card"
+// DC == “debit card”
```
Comment (fionawhim) [inline services-js/registry-certs/lib/costs.ts:13]: lol

**Review by fionawhim — APPROVED** [review]

File: services-js/registry-certs/lib/costs.ts (hunk 3 above)
Comment (jessicamarcus) [reply services-js/registry-certs/lib/costs.ts:13]: 😉

**Pull request merged**